src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -53,6 +53,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <EventPipeName Condition="'$(EventSourceSupport)' == 'true'">libeventpipe-enabled</EventPipeName>
 
       <LinkStandardCPlusPlusLibrary Condition="'$(LinkStandardCPlusPlusLibrary)' == '' and '$(_IsiOSLikePlatform)' == 'true' and '$(InvariantGlobalization)' != 'true'">true</LinkStandardCPlusPlusLibrary>
+      <ClrgcSupportName>libclrgc-disabled</ClrgcSupportName>
+      <ClrgcSupportName Condition="'$(IlcEnableClrgc)' == 'true'">libclrgc-enabled</ClrgcSupportName>
```

Comment:
`EnableClrgc` suggests that the GC enabled by default is not CLR GC, that does not sound right.

We call this [standalone GC in the doc](https://learn.microsoft.com/en-us/dotnet/core/runtime-config/garbage-collector#standalone-gc), so this setting should use the same name. Something like: `IlcStandaloneGCSupport`.